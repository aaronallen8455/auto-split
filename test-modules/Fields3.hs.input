import qualified Import as I

main :: IO ()
main = pure ()

test :: I.R
test = FIELDS I.R { }
