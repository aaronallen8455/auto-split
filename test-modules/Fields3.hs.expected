import qualified Import as I

main :: IO ()
main = pure ()

test :: I.R
test = I.R { I.r1 = _, I.r2 = _ }
